homu/html/queue.html

```diff
@@ -222,9 +234,38 @@ <h1>Homu queue - {% if repo_url %}<a href="{{repo_url}}" target="_blank">{{repo_
             ];
 
             document.getElementById('expand-rollup').onclick = function() {
-                var checkboxCount = document.querySelectorAll('#queue tbody input[type=checkbox]:checked').length;
+                var els = document.querySelectorAll('#queue tbody input[type=checkbox]:checked');
+                var checkboxCount = els.length;
                 document.getElementById('checkbox-count').innerHTML = checkboxCount;
                 document.getElementById('actual-rollup').className = '';
+
+                var value = 0;
+                for (var i = 0; i < els.length; i++) {
+                    var rollupClassName = els[i].parentNode.parentNode.children[10].className;
+                    if (rollupClassName === "rollup_always") {
+                    } else if (rollupClassName === "rollup_iffy") {
+                        value += 2;
+                    } else { // nothing or "rollup_maybe"
+                        value += 1;
+                    }
+                }
+                value = (value / 8) + (Math.max((checkboxCount - 8), 0) / 8);
+
+                var riskiness = 'not risky';
+                var riskinessClass = 'riskiness_low';
+                if (value > 0 && value <= 0.5) {
+                    riskiness = 'acceptably risky';
+                } else if (value <= 1.0) {
+                    riskiness = 'somewhat risky';
+                } else if (value <= 2.0) {
+                    riskinessClass = 'riskiness_medium';
+                    riskiness = 'risky';
+                } else if (value > 2.0) {
+                    riskinessClass = 'riskiness_high';
+                    riskiness = '⚠️very risky⚠️';
+                }
```

Comment:
These limits are super low, and I'm worried that since they'll trigger on almost every PR people will just ignore them. Here's some rollups that landed in the last few days:
- https://github.com/rust-lang/rust/pull/84005: 5 PRs, riskiness (1 + 0 + 1 + 0 + 1 = 3) = "very risky", CI failed
- https://github.com/rust-lang/rust/pull/83986: 5 PRs, riskiness (1 + 1 + 1 + 0 + 0 = 3) = "very risky", CI passed
- https://github.com/rust-lang/rust/pull/83964: 8 PRs, riskiness (0 + 1 + 1 + 1 + 1 + 0 + 0 + 1 = 5) = "very risky", CI passed

None of these PRs are anything less than "very risky"; I doubt you'll find any that are in the past few weeks (there's a list at https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3Arollup; finding the riskiness is manual).

Maybe as an alternative you could raise the limit to something like 4, and mark `rollup=maybe` as a value of 3 instead? That seems closer to how rollups are done now.

Reply:
Honestly even the rollups I linked don't seem super risky to me, I would mark the 8 PR rollup as "riskiness_medium" and the rest as "riskiness_low". "riskiness_high" would be rollups with over 12 PRs (and I have some monsters with over 20 I could link too).